app/assets/stylesheets/ama_layout/layout_components/forms.scss

```diff
@@ -150,19 +150,21 @@ input[type="checkbox"]{
   }
 }
 
-.month,
-.year{
-  @extend %inputs-thirds;
-}
+.date{
```

Comment:
can you just throw a comment in here to mention these classes are nested to prevent conflict with foundations datepicker?